src/functions-and-arrays.js

```diff
@@ -52,14 +105,31 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(words) {
+  let sortedWords = []
+    for(i = 0; i<words.length; i++) {
+        if(words[i] != sortedWords){
+          sortedWords.push(words[i])
+        }
+        
+  }
+  return sortedWords
+}
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(wordsFind) {
+  let searchWord = "matter"
+  for(i = 0; i<wordsFind.length; i++) {
+    if(wordsFind[i] === searchWord){
+      return true   
+    }
+  }
+  return false
+}
```

Comment:
Hier hast du einen Parameter vergessen. Wir müssen ein Wort übergeben und ein Array. Erst das Array und dann das Wort nach dem wir in dem Array suchen. Also anstatt eine Variable zu erstellen, muss es so sein:

```
function doesWordExist(arrOfWords, wordToSearch) {

  // conditional for passing test if array is empty
  if (arrOfWords.length === 0) {
    return null;
  }

  // a loop that will check each for of the array against the wordSearch. return true only if it encounters it.
  for (let i = 0; i < arrOfWords.length; i++) {
    if (arrOfWords[i] === wordToSearch) {
      return true // if at some point the word is found, they function will return true and stop executing the rest of the function.
    }
  }

  return false // if the loop runs and it does not encounters the words (doesn't return true) then it will return false.

```